apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -254,3 +254,34 @@ If you want to diagnose session management issues,
 the statistics for the session pool associated with the connection:
 
 {{sample:SpannerConnection.GetSessionPoolSegmentStatistics}}
+
+## RpcException from SpannerException
+
+The `RpcException` property of `SpannerException` returns the underlying
+`RpcException` which caused the `SpannerException`. The returned 
+value may be null if the `SpannerException` was not caused by an `RpcException`.
+
+`ErrorInfo` in [RpcException](../../../docs/devsite-help/troubleshooting.md#how-can-i-get-errorinfo-from-rpcexception) can 
+provide actionable error deatils. 
+
+`RpcException` property of `SpannerException` can be used like:
+
+```csharp
+using Google.Api.Gax.Grpc;
+try
+{
+    // Code using Spanner. 
+}
+catch (SpannerException ex)
+{
+    var rpcException = ex.RpcException;
+    if (rpcException != null)
+    {
+        // Use the domain, reason and metadata of rpcException's ErrorInfo, e.g. for logging or taking remedial action.
```

Comment:
Hmm... there are a couple of options here. We could show three scenarios:

- RpcException with ErrorInfo (including how to get it)
- RpcException without ErrorInfo
- No RpcException

Or we could just show RpcException with ErrorInfo, and no RpcException.

Reply:
Updated sample for all three scenarios.